lib/codegen.js

```diff
@@ -53,6 +53,11 @@ var getViewForSwagger2 = function(opts, type){
             };
             op.parameters = op.parameters ? op.parameters : [];
             op.parameters.forEach(function(parameter) {
+                if ('$ref' in parameter) { 
```

Comment:
In the base swagger object you can specify reusable parameters (https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#user-content-swaggerParameters) and then use $ref to reference them.

I have a rest api, and for example there are edit and delete functions that require and id in the url (e.g. http://api.com/comments/1). I specify an id parameter in the previous stated reusable parameters and then reference them using $ref.